openmetadata-docs/content/openmetadata/connectors/ml-model/mlflow/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Mlflow
+slug: /openmetadata/connectors/ml-model/mlflow
+---
+
+<ConnectorIntro service="ml-model" connector="Mlflow"/>
+
+<Requirements />
+
+<MetadataIngestionService connector="Mlflow"/>
+
+<h4>Connection Options</h4>
+
+- **trackingUri**: Mlflow Experiment tracking URI. E.g., http://localhost:5000
+- **registryUri**: Mlflow Model registry backend. E.g., mysql+pymysql://mlflow:password@localhost:3307/experiments
+
+<IngestionScheduleAndDeploy />
```

Comment:
Here, we will need to build a new component like `MlModelIngestionConfig` just like we did for Databases and other sources

Reply:
You can refer to this [pr](https://github.com/open-metadata/docs/pull/155)